### PR TITLE
scraper for ky hart county

### PR DESCRIPTION
**Pull request by eitalse**
scraper for ky hart county
pretty standard jail tracker just added some enum mapping

### CODE REVIEW

File: recidiviz/ingest/scrape/regions/us_ky_hart/us_ky_hart_scraper.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        """
+        return 'HART_COUNTY_KY'
+
+    def get_enum_overrides(self):
+        return {'P OR STAY': BondType.CASH,
+                'T': ChargeClass.PROBATION_VIOLATION,
+                'UNKOWN': Race.EXTERNAL_UNKNOWN,
+                'V': ChargeClass.INFRACTION,
+                'VIOLATION': ChargeClass.INFRACTION,
+                '10% CASH': BondType.PARTIAL_CASH,
```
Comment (jamwalla) [inline recidiviz/ingest/scrape/regions/us_ky_hart/us_ky_hart_scraper.py]: Does this include the changes from #814? I think you don't need `10% CASH` anymore, the `10 CASH` below will catch both.

File: region_manifest.yaml [hunk 2]
```diff
@@ -683,6 +683,14 @@ regions:
...
     scraper_package: us_ks_pratt
     timezone: America/Chicago
+  us_ky_hart:
+    agency_name: Hart County Sheriff
+    region_code: us_ky_hart
+    agency_type: jail
+    queue: us-ky-hart-scraper
+    base_url: http://www.hartcountykyjail.com/InmateRoster/HCJ_inmatelist.html
+    scraper_package: us_ky_hart
+    timezone: America/New_York
```
Comment (jamwalla) [inline region_manifest.yaml]: news to me, but kentucky is split across timezones! So this should be `America/Chicago`.

File: recidiviz/ingest/scrape/regions/us_ky_hart/us_ky_hart_scraper.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        A JailTracker landing page URL ends with: "/jailtracker/index/<INDEX>".
+        This value can either be text or a number. In either case, this method
+        should return the value as a string.
+        """
+        return 'HART_COUNTY_KY'
+
+    def get_enum_overrides(self):
+        return {'P OR STAY': BondType.CASH,
+                'T': ChargeClass.PROBATION_VIOLATION,
+                'UNKOWN': Race.EXTERNAL_UNKNOWN,
```
Comment (jamwalla) [inline recidiviz/ingest/scrape/regions/us_ky_hart/us_ky_hart_scraper.py]: their misspelling?

Comment (eitalse) [reply recidiviz/ingest/scrape/regions/us_ky_hart/us_ky_hart_scraper.py]: yes

File: region_manifest.yaml (hunk 2 above)
Comment (eitalse) [reply region_manifest.yaml]: kk

File: recidiviz/common/constants/person.py [hunk 4]
```diff
@@ -97,6 +97,7 @@ def _get_default_map():
     'WHITE': Race.WHITE,
     'WHITE EURP N AFR MID EAS': Race.WHITE,
     'WHITE EURP N AFR': Race.WHITE,
+    'BIRACIAL': Race.OTHER,
```
Comment (jessex) [inline recidiviz/common/constants/person.py]: This should go up with the other `Race.OTHER` values, on line 91.

File: recidiviz/ingest/scrape/regions/us_ky_hart/us_ky_hart_scraper.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        region's landing page.
+
+        A JailTracker landing page URL ends with: "/jailtracker/index/<INDEX>".
+        This value can either be text or a number. In either case, this method
+        should return the value as a string.
+        """
+        return 'HART_COUNTY_KY'
+
+    def get_enum_overrides(self):
+        return {'P OR STAY': BondType.CASH,
```
Comment (jessex) [inline recidiviz/ingest/scrape/regions/us_ky_hart/us_ky_hart_scraper.py]: Generally, we drop the first key-value pair in an in-line dictionary initialization to the line after the opening curly brace.

**Pull request merged**